test/Acme/ExampleClass.php

```
@@ -177,3 +177,34 @@ public static function someStaticFunc()
+    public function falsy($int = 0, $float = 0.0, $bool = false, $none = null)
+    {
+    }
+}
+
+class ClassWithFluentInterface
+{
+
+    /**
+     * @return self
+     */
+    public function fluent()
```

PHPStan (2.2.6)

[error] Method Acme\ClassWithFluentInterface::fluent() should return Acme\ClassWithFluentInterface but return statement is missing.


(return.missing)